### PR TITLE
docs: adding new logo

### LINKED ISSUE #101: chore: design a new logo for the project

### PR DESCRIPTION
**Pull request by JessicaSachs**
This closes #101 :-)

* Updates the README and makes it prettier (including adding logos)
* Adds logo to website
* Adds new logo creative to repo
* Add GH Stars link to website

I’ll put some of the twitter assets here, but they’re not getting committed.

**Logo + Favicon**
9717F0EE-EAE4-4EDB-BE0D-84CDF92108C7

**Twitter Banner**
5D49DC61-FEAC-4BA7-BD26-78A1BF380600

**Fonts**
Header: Sukar Black
Sans Serif: Raleway
Mono: Fira Code

I need help to add dark mode back into the favicon. #195. I tried to make some (bad) ones. Help from someone would be lovely.
C97814C0-F5CC-41EB-ACB5-DBA9DC242940

### CODE REVIEW

**Review by import-brain — DISMISSED** [review]

**Comment by Shinigami92** [comment]
I hope it's not to late and not to much work, but can we move it a bit to the right? Feels a bit offset :sweat_smile: 
image

Twitter looks good IMO :heavy_check_mark: 

image

**Comment by JessicaSachs** [comment]
Yeah I agree. Even though it’s technically even, it doesn’t look right. Let
me fix.

File: README.md [hunk 1]
```diff
@@ -13,16 +17,24 @@ Please replace your `faker` dependency with `@faker-js/faker`. This is the offic
 npm install @faker-js/faker --save-dev
 ```
 
-or
+### Browser
```
Comment (Shinigami92) [inline README.md:20]: Looks like there are many unrelated changes, do we need a rebase?

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JessicaSachs) [reply README.md:20]: These aren’t unintended. I cleaned up the README as part of adding the logo and stuff.

**Comment by JessicaSachs** [comment]
The assets are actually centered. The Discord one may need to be re-uploaded. I think this is good to merge.

File: README.md (hunk 1 above)
Comment (Shinigami92) [reply README.md:20]: But you remove the yarn and pnpm install instructions :eyes: 
We just added them one day ago or so by intention

Could you move the needed cleanup into its own PR? :slightly_smiling_face:

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

**Comment by JessicaSachs** [comment]
I know. I was the one who added them, too, but that was before we had docs website and afterwards the README just didn’t look good.

File: README.md [hunk 2]
```diff
@@ -13,308 +17,76 @@ Please replace your `faker` dependency with `@faker-js/faker`. This is the offic
...
-  - color
-  - vrm
-  - bicycle
-
-### Faker.fake()
-
-Faker contains a super useful generator method `Faker.fake` for combining faker API methods using a mustache string format.
-
-**Example:**
+Types are available via the `@types/faker` package. You must manually link them using a `*.d.ts` file, like so:
```
Comment (Shinigami92) [inline README.md:54]: This is also outdated for v6.0.0-alpha.3 :thinking:

File: README.md [hunk 3]
```diff
@@ -13,308 +17,76 @@ Please replace your `faker` dependency with `@faker-js/faker`. This is the offic
...
+| Name     	| `faker.name.firstName()`       	| Cameron                                                                                                                                                                                                   	|
+| Phone    	| `faker.phone.phoneNumber()`    	| +1 291-299-0192                                                                                                                                                                                           	|
+| System   	| `faker.system.directoryPath()` 	| C:\Documents\Newsletters\                                                                                                                                                                                 	|
+| Vehicle  	| `faker.vehicle.vehicle()`      	| 2011 Dodge Caravan                                                                                                                                                                                        	|
+
+### Localization
 
 As of version `v2.0.0` Faker has support for multiple localities.
```
Comment (Shinigami92) [inline README.md:91]: Should we the version here?
So just: `Faker has support for multiple localities.`

File: README.md [hunk 4]
```diff
@@ -394,21 +117,38 @@ If you want consistent results, you can set your own seed:
...
+### Browser Bundle
+
+```shell
+npm run browser
+```
+
+### Testing
+
+```shell
+npm install .
```
Comment (Shinigami92) [inline README.md:145]:
```suggestion
npm install
```

I assume the period was not intended, was it?

---

Beside that, I think currently you need to also run `npm run build` before tests
This could change soon

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (JessicaSachs) [reply README.md:54]: I can just remove it, right?

Comment (JessicaSachs) [reply README.md:54]: Won't remove it until v6 is released.

File: README.md (hunk 3 above)
Comment (JessicaSachs) [inline README.md:91]:
```suggestion
Faker has support for multiple locales.
```

**Review by Shinigami92 — APPROVED** [review]

**Review by import-brain — APPROVED** [review]

**Review by clarkerican — APPROVED** [review]
